src/services/ReadModelGateway.ts

```diff
@@ -0,0 +1,16 @@
+/*  =======================================================================  
+  IMPORTANT: This service mocks all operations performed throught read models 
+===========================================================================  */
+
+import { EService } from "../model/domain/models.js";
+
+export interface IReadModelGateway {
+  readonly getEServiceByName: (name: string) => Promise<EService>;
+  readonly getOrganizationID: () => Promise<string>;
+}
+
+export const ReadModelGateway: IReadModelGateway = {
+  getEServiceByName: async (_name: string): Promise<EService> => undefined,
```

Comment:
Questo mi ha molto confuso perché il tipo di ritorno non sembra includere `undefined`, poi ho visto che `EService` stesso contiene `undefined` in seguito a una modifica che mi era sfuggita nella PR prima.

Secondo me `EService` *non* deve includere `undefined`, perché è un concetto di dominio, non ha senso che la definizione di un `EService` includa l'assenza dell'`EService` stesso. 😄 

Quindi toglierei `| undefined` dalla definizione di `EService` e lo aggiungerei qui nel tipo di ritorno (e altrove se serve altrove):
```suggestion
  getEServiceByName: async (_name: string): Promise<EService | undefined> => undefined,
```
(credo fosse quello che intendeva @gabro, se no parliamone).

Reply:
Confermo, mi torna tutto quello che dice @tpetrucciani.

Reply:
Tra l'altro, forse minor, ma mi torna di più usare `null` piuttosto che `undefined` per indicare l'assenza di un valore (cambia poco, ma `undefined` è implicitamente ritornato anche quando manca un `return` mentre `null` è esplicito, quindi forse più adatto)

Reply:
mi torna il tutto, ha perfettamente senso